aiidalab_widgets_base/computational_resources.py

```diff
@@ -871,10 +883,19 @@ def _configure_computer_ssh(self, computer: orm.Computer, user: orm.User):
                 message = "SSH username is not provided"
                 raise RuntimeError(message) from exc
 
-        if "proxycommand" in sshcfg:
-            authparams["proxy_command"] = sshcfg["proxycommand"]
-        elif "proxyjump" in sshcfg:
-            authparams["proxy_jump"] = sshcfg["proxyjump"]
+        if "proxy_jump" in self.computer_configure:
+            authparams["proxy_jump"] = self.computer_configure["proxy_jump"]
+
+        if "proxy_command" in self.computer_configure:
+            authparams["proxy_command"] = self.computer_configure["proxy_command"]
+
+        if "key_filename" in self.computer_configure:
+            authparams["key_filename"] = os.path.expanduser(
+                self.computer_configure["key_filename"]
+            )
```

Comment:
Small nitpick. I am trying to convert to Pathlib everywhere. If you could do something like [this](https://stackoverflow.com/questions/57248819/how-to-use-pathlib-path-expanduser-and-amend-and-use-a-posixpath-value) here - it would be awesome.

Reply:
There is another occurrence of `os.path.expanduser` in this file. Please change there too.

Reply:
Yes, changed. I agree that moving to using `Path` is better.